lib/tasks/database_data.rake

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+namespace :database_data do
+  # These tasks are for getting production data into local, qa, and stage environments
+  # :get_prod_data_file should be run first
+  # Then run :load_to_local, :load_to_qa, or :load_to_stage to load the data into
+  # your local, qa, or stage environments respectively
+
+  task get_prod_data_file: :environment do
+    desc 'Pull production data down as a sql.gz file and store it in the tmp directory'
+    hostname = 'rmdweb1prod'
+
+    # Pull down db config to be parsed
+    db_config = YAML.safe_load(`ssh deploy@#{hostname} -p 1855 'cat rmd/current/config/database.yml'`)
+
+    # Parse values from db config
+    db_password = db_config['production']['password']
+    db_username = db_config['production']['username']
+    db_host = db_config['production']['host']
+    db_name = db_config['production']['database']
+
+    # Dump data into sql file and gzip
+    `ssh deploy@#{hostname} -p 1855 PGPASSWORD=#{db_password} 'pg_dump --clean \
+                                                                       --no-owner \
+                                                                       -U #{db_username} \
+                                                                       -h #{db_host} \
+                                                                       -d #{db_name} > psql-rmd-prod-data.sql;
+                                                               gzip psql-rmd-prod-data.sql'`
```

Comment:
I reckon you ought to timestamp the file on the server, just in case two developers happen to run this at the same time. Since this will likely take a few minutes to run, it's not entirely impossible

Reply:
And riffing on the same "this will take a while" note, I do think it would be user friendly to `puts` some status updates along the way, so we can tell that it's doing something. Like maybe even splitting up the pgdump from the gzip, so that you can report the progress